cea/globalvar.py

```diff
@@ -266,18 +120,17 @@ def __init__(self):
         self.PipeCostPerMeterAnnual = self.PipeCostPerMeterInv / self.PipeLifeTime
         self.NetworkDepth = 1  # m
 
-        # Solar area to Wpeak
-        self.eta_area_to_peak = 0.16  # Peak Capacity - Efficiency, how much kW per area there are, valid for PV and PVT (after Jimeno's J+)
 
-        # Pressure losses
-        # self.DeltaP_DCN = 1.0 #Pa - change
-        # self.DeltaP_DHN = 84.8E3 / 10.0 #Pa  - change
 
-        self.PumpEnergyShare = 0.01  # assume 1% of energy required for pumping, after 4DH
-        self.PumpReliabilityMargin = 0.05  # assume 5% reliability margin
+        self.CC_Maintenance_per_kWhel = 0.03 * self.EURO_TO_CHF  # 0.03 € / kWh_el after Weber 2008, used in Slave Cost Calculation
 
-        # Circulating Pump
-        self.etaPump = 0.8
+        self.ELEC_PRICE = 0.2 * self.EURO_TO_CHF / 1000.0  # default 0.2
```

Comment:
I believe all this variables related to prices should either be part of the cost database or the config file. 
you can add one variable called conversion_factor. which basically converts all the values from your database to whatever currency you want.